### PR TITLE
Styled Tooltip

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**
- [X] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/mMYjbJyj

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Tooltip/Tooltip.test.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { Tooltip } from '.';
+
+describe('Tooltip', () => {
+  const testId = 'tooltip';
+  it('should render trigger', () => {
+    const trigger = 'trigger';
+    const tooltip = 'tooltip';
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Tooltip/Tooltip.test.tsx:9]: move this one level up and remove from all the test cases

File: packages/styled-components/src/components/primitives/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { usePopper } from 'react-popper';
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Tooltip/Tooltip.tsx:3]: this has to be added as a dependency, am I right?

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
   "scripts": {
-    "storybook": "echo 'Starting Storybook' && start-storybook -p 8080 -c .storybook",
+    "storybook": "echo 'Starting Storybook' && start-storybook -p 8080 -c .storybook --no-dll",
```
[the remark this answers is not in the document]
Comment (guilean) [reply packages/styled-components/package.json:16]: Due to an issue with react-popper/storybook https://github.com/storybookjs/storybook/issues/10982

File: packages/styled-components/src/components/primitives/Tooltip/Tooltip.tsx (hunk 2 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Tooltip/Tooltip.tsx:3]: As far as I remember we didn't remove these dependencies from package.json of styled-components (I can see this dependency on line 32)

File: packages/styled-components/src/components/primitives/Tooltip/Tooltip.styled.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    height: 0.5rem;
+    z-index: 10;
+    content: '';
+    transform: rotate(45deg);
+  }
+
+  &.primary900::before {
+    background-color: ${(props) => getTheme(props).colors.fillPrimary900};
+  }
+  &:before &.primary100::before {
```
[the remark this answers is not in the document]
Comment (guilean) [reply packages/styled-components/src/components/primitives/Tooltip/Tooltip.styled.ts:39]: Oops.. removed.

File: packages/styled-components/src/components/primitives/Tooltip/Tooltip.test.tsx (hunk 1 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Tooltip/Tooltip.test.tsx:9]: Done

File: packages/styled-components/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (tkanzakic) [reply packages/styled-components/package.json:16]: is this fixed in the latest release https://github.com/storybookjs/storybook/issues/10982#issuecomment-715255106?

[the remark this answers is not in the document]
Comment (guilean) [reply packages/styled-components/package.json]: Updated to latest version of storybook and removed `--no-dll` (no longer needed)

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**